global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
-    "version": "7.0.404",
-    "rollForward": "latestFeature",
```

Comment:
`latestFeature` should be allowed as an option, otherwise it is crazy to need to update this file to test a new sdk version